connector/src/main/java/org/geysermc/connector/network/translators/item/ItemRegistry.java

```diff
@@ -63,6 +63,8 @@
     public static final int SHIELD = 829;
     // Boat ID, used in BedrockInventoryTransactionTranslator.java
     public static final int BOAT = 333;
+    // Bucket ID, used in BedrockInventoryTransactionTranslator.java
+    public static final int BUCKET = 325;
```

Comment:
It's fine for now since it's out of the scope of this PR, but these should probably be organized alphabetically or by number.